src/Core/src/Handlers/View/ViewHandler.Windows.cs

```diff
@@ -133,17 +133,6 @@ internal static void MapContextFlyout(IElementHandler handler, IContextFlyoutEle
 			}
 		}
 
-		public virtual bool NeedsContainer
-		{
-			get
-			{
-				if (VirtualView is IBorderView border)
```

Comment:
The windows check here for `NeedsContainer` doesn't include "clip" or "shadow" which seems like an oversight because the `ViewHandler` was forcing a container inside MapClip